src/components/atoms/__tests__/AppCheckbox.spec.js

```diff
@@ -0,0 +1,19 @@
+import {shallowMount} from '@vue/test-utils'
+import AppCheckbox from '@/components/atoms/AppCheckbox.vue';
+
+describe('AppCheckbox.vue', () => {
+    it('Should render AppCheckbox with default props', () => {
+        const wrapper = shallowMount(AppCheckbox);
+        expect(wrapper.element).toMatchSnapshot();
+    });
+
+    it('Should render AppCheckbox with props set', () => {
+        const wrapper = shallowMount(AppCheckbox, {
```

Comment:
You can use setProps to change the properties of your component, no need to call shallowMount again